code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -62,8 +62,8 @@
 
 /obj/item/reagent_containers/food/snacks/roburger
 	name = "roburger"
-	desc = "The lettuce is the only organic component. Beep."
-	icon_state = "roburger"
+	desc = "Looking at this makes your flesh feel like weakness."
+	icon_state = "Hamborger"
```

Comment:
You accidentally changed the icon state not the object name.

Reply:
Ah yes thank you! This is my first time submitting a change and apparently missed that crucial detail! Appreciate the lookover